src/address.rs

```diff
@@ -40,31 +24,65 @@ impl Address {
     /// creates the zeros. In our case we treat it as "empty"
     /// address, which is then reperesented by zeros.
     pub fn new() -> Address {
-        Address { data: Vec::new() }
+        Address([0; 20])
```

Comment:
It's the same story as `PrivateKey`. What's the point of a `new` that doesn't yield a correct address? The bytes of the address can't even be changed. If someone wanted a 0-filled address they should just use the `From<[u8; 20]>` trait.

Reply:
Saved for prosperity #57